### PR TITLE
License under AGPLV3

### PR DESCRIPTION
**Pull request by Waldz**
Closes: #54
Closes: https://github.com/mysteriumnetwork/node/issues/1843

Licensing under same license as https://github.com/OpenVPN/openvpn3

### CODE REVIEW

File: examples/desktop/main.go [hunk 1]
```diff
@@ -1,18 +1,20 @@
 /*
- * Copyright (C) 2018 The "MysteriumNetwork/go-openvpn" Authors.
+ * go-openvpn -- Go gettable library for wrapping Openvpn functionality in go way.
+ *
+ * Copyright (C) 2020 The "MysteriumNetwork/go-openvpn" Authors..
```
Comment (chompomonim) [inline examples/desktop/main.go:4]: Here should be Copyright (C) 2020 Blockdev AG

File: examples/mobile/application.go [hunk 2]
```diff
@@ -1,18 +1,20 @@
 /*
- * Copyright (C) 2018 The "MysteriumNetwork/go-openvpn" Authors.
+ * go-openvpn -- Go gettable library for wrapping Openvpn functionality in go way.
+ *
+ * Copyright (C) 2020 The "MysteriumNetwork/go-openvpn" Authors..
```
Comment (chompomonim) [inline examples/mobile/application.go:4]: Same as above `Copyright (c) 2020 BlockDev AG`

**Review by chompomonim — COMMENTED** [review]
We should everywhere use "Copyright (C) 2020 Blockdev AG" instead of "Copyright (C) 2020 The "MysteriumNetwork/go-openvpn" Authors.."

File: examples/desktop/main.go (hunk 1 above)
Comment (Waldz) [reply examples/desktop/main.go]: Fixed

**Review by chompomonim — APPROVED** [review]

**Pull request merged**